# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/webcodecs/video-decoder.crossOriginIsolated.https.any.js.ini
# Commit: 200c174fc2d1
# Full Hash: 200c174fc2d1497eb5327f84cc3a4efc90419179
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2023-06-29 15:31:45
# Regressor Bug: 1831451
# File Overlap Count: 1
# Description:
#   Bug 1831451 - Enable VideoDecoder on Linux only r=padenot
#   
#   The codec support will be done in Bug 1840508 later
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D181917
# ==============================================================================

diff -r cd30b650aeed -r 200c174fc2d1 testing/web-platform/meta/webcodecs/video-decoder.crossOriginIsolated.https.any.js.ini
--- a/testing/web-platform/meta/webcodecs/video-decoder.crossOriginIsolated.https.any.js.ini	Thu Jun 29 04:15:04 2023 +0000
+++ b/testing/web-platform/meta/webcodecs/video-decoder.crossOriginIsolated.https.any.js.ini	Thu Jun 29 04:15:05 2023 +0000
@@ -3,8 +3,28 @@
   expected:
     if (os == "android") and fission: [OK, TIMEOUT]
 
+  [Test isConfigSupported() and configure() using a SharedArrayBuffer]
+    expected:
+      if (os != "linux"): [PRECONDITION_FAILED]
+      PASS
+
+  [Test isConfigSupported() and configure() using a Uint8Array(SharedArrayBuffer)]
+    expected:
+      if (os != "linux"): [PRECONDITION_FAILED]
+      PASS
+
 
 [video-decoder.crossOriginIsolated.https.any.worker.html]
   prefs: [dom.media.webcodecs.enabled:true]
   expected:
     if (os == "android") and debug and not swgl: [OK, TIMEOUT]
+
+  [Test isConfigSupported() and configure() using a SharedArrayBuffer]
+    expected:
+      if (os != "linux"): [PRECONDITION_FAILED]
+      PASS
+
+  [Test isConfigSupported() and configure() using a Uint8Array(SharedArrayBuffer)]
+    expected:
+      if (os != "linux"): [PRECONDITION_FAILED]
+      PASS